module/entities/helper.js

```diff
@@ -1187,8 +1201,119 @@ module.exports = class EntitiesHelper {
         }
     }
 
+     /**
+   * Push entities data to elastic search
+   * @method
+   * @name pushEntitiesToElasticSearch
+   * @name entityId - created or modified entity data.
+   * @returns {Object} 
+   */
+
+  static pushEntitiesToElasticSearch(entityId) {
+    return new Promise(async (resolve, reject) => {
+        try {
+
+            let telemetry = [];
+            
+            let entityDocuments = await this.entityDocuments({
+                    _id: {
+                      $in : entityId
+                    }
+                }, [
+                        "_id",
+                        "metaInformation",
+                        "entityType",
+                        "entityTypeId",
+                        "updatedAt",
+                        "createdAt",
+                    ]);
+            
+            await Promise.all(entityDocuments.map(async entityDocument => {
```

Comment:
@deepakr123 - Do it one by one and not in parallel

Reply:
resolved